Formula/c/cortexso.rb

```diff
@@ -0,0 +1,38 @@
+require "language/node"
+
+class Cortexso < Formula
+  desc "Drop-in, local AI alternative to the OpenAI stack"
+  homepage "https://jan.ai/cortex"
+  url "https://github.com/janhq/cortex.git",
+      tag:      "v0.4.16",
+      revision: "edd3d8d1ae20cb11a6a5cd86e7a2796a7fa8d4df"
+  license "Apache-2.0"
+  head "https://github.com/janhq/cortex.git", branch: "dev"
+
+  depends_on "node" => :build
+  depends_on "python-setuptools" => :build
+  depends_on "python@3.12" => :build
+  depends_on :macos
+  depends_on macos: :ventura
```

Comment:
Why?

Reply:
Hi @SMillerDev, I'm checking why the CI test failed, now removing it.

Reply:
Somehow bottle build is broken on Linux, it runs compiler commands instead of the binary commands. I did try decompress the `.bottle.tar.gz` and the binary works fine. Only when `brew install .bottle.tar.gz`

Reply:
> Somehow bottle build is broken on Linux, it runs compiler commands instead of the binary commands.

Does this install pre-built binaries then?

Reply:
> > Somehow bottle build is broken on Linux, it runs compiler commands instead of the binary commands.
> 
> Does this install pre-built binaries then?

Hi @carlocab,

It does, but the final executable file acts weirdly, showing Bun compiler commands. Sus, install / untar process somehow breaks the eggshell. I tried decompressing bottle.tar.gz manually and replacing the installed file. It does work.